src/Language/PureScript/AST/Binders.hs

```diff
@@ -61,7 +63,9 @@ data Binder
   -- A binder with a type annotation
   --
   | TypedBinder SourceType Binder
-  deriving (Show)
+  deriving (Show, Generic)
+
+instance NFData Binder
```

Comment:
Here and generally: can you consistently produce `NFData` instances using `deriving`, using `DeriveAnyClass` and [`DerivingStrategies`](https://ghc.gitlab.haskell.org/ghc/doc/users_guide/exts/deriving_strategies.html) (already in our default extensions list) as needed to enable/disambiguate this?